gno.land/pkg/gnoweb/gnoweb_test.go

```diff
@@ -27,15 +27,15 @@ func TestRoutes(t *testing.T) {
 		{"/", ok, "Welcome"}, // assert / gives 200 (OK). assert / contains "Welcome".
 		{"/about", ok, "blockchain"},
 		{"/r/gnoland/blog", ok, ""}, // whatever content
-		{"/r/gnoland/blog?help", ok, "exposed"},
+		{"/r/gnoland/blog$help", ok, "exposed"},
 		{"/r/gnoland/blog/", ok, "admin.gno"},
 		{"/r/gnoland/blog/admin.gno", ok, "func "},
 		{"/r/demo/users:administrator", ok, "address"},
 		{"/r/demo/users", ok, "manfred"},
 		{"/r/demo/users/users.gno", ok, "// State"},
 		{"/r/demo/deep/very/deep", ok, "it works!"},
 		{"/r/demo/deep/very/deep:bob", ok, "hi bob"},
-		{"/r/demo/deep/very/deep?help", ok, "exposed"},
```

Comment:
can you add a few tests with at least:

- `/r/gnoland/blog$help&func=Existing`
- `/r/gnoland/blog$help&func=NonExisting`
- `/r/gnoland/blog?arg1=val1&arg2=val2`

Reply:
Changeset d974783